scripts/src/init_build.rs

```diff
@@ -1,23 +1,22 @@
-use anyhow::Result;
+use anyhow::{Context, Result};
 use ck3oop_scripts::build;
+use log::{info, warn};
+
 fn main() -> Result<()> {
     let dir = build::get_full_path_to_build_dir()?;
     if !dir.exists() {
         std::fs::create_dir_all(dir.clone())?;
-        return Ok(println!(
-            "Created build directory at: {:?}",
-            dir.clone().into_os_string()
-        ));
+        info!("Created build directory at: {:?}", dir.clone().into_os_string());
+        return Ok(());
```

Comment:
we should not return here (cannot create file after dir)

Reply:
![image](https://github.com/bukowa/ck3oop/assets/206368/2363cac2-a009-49ee-90dc-6f42ebf4e071)